packages/debug/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@matanuska/debug",
+  "version": "1.0.0",
+  "description": "Matanuska debug library",
+  "type": "module",
+  "main": "index.js",
+  "scripts": {
+    "build": "tsc -p tsconfig.build.json",
```

Comment:
Given this exports a single file, it *may* make sense to build this one with grabthar.